content/en/docs/score specification/score-spec-reference.md

```diff
@@ -14,12 +14,12 @@ aliases:
 
 The Score Specification is a YAML file that contains the following top-level reference definitions.
 
-Use these definitions to describe a single Workload.
+Use these definitions to describe a single workload.
 
-- [Workload definition](#workload-definition): (required) defines the metadata and `apiVersion`.
-- [containers](#container-definition): (required) defines how the Workload's tasks are executed.
-- [resources](#resources-definition): (optional) defines dependencies needed by the Workload.
-- [service](#service-definition): (optional) defines how an application can expose its resources when executed.
+- [Workload definition](#workload-definition): (required) defines the API version and metadata.
+- [service](#service-definition): (optional) defines the service that the workload provides.
```

Comment:
I'd suggest we still explain `service` one _after_ the `containers` section. That should increase familiarity because most people will feel more comfortable seeing the container definition first - it's what people expect and loks familiar.

I'm not sure why we define service first in the spec document, but the order there isn't particularly purposeful.

Reply:
gotcha, yes I was orienting towards the order in the schema. Will change it back and we can eventually change it in the schema for consistency. changed it back 👍